deploy/ics-operator/manifests/provider-role.yaml

```diff
@@ -47,7 +47,7 @@ rules:
 - apiGroups:
   - ocs.openshift.io
   resources:
-  - storageclassclaims
+  - storageclassrequests
```

Comment:
Everything under deploy is a generated change, it should be in commit 2

Reply:
@umangachapagain We should be keeping these IBM bundle changes DS only now, right?

Reply:
yes that is right. We will remove them maybe in 4.14.